src/cli.ts

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
Removing this actually broke CI because it doesn't know this is a node executable given you're transforming it into JS and you can't simply run JS from bash (whatever shell).

If you put it back the build should pass :)

Reply:
Oops copy mistake, Will fix it. Thanks for catching it 👍🏽